Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/StubIrBuilder.kt

```diff
@@ -257,7 +257,7 @@ class StubsBuildingContextImpl(
     override val wrapperComponentsBuilder = WrapperGenerationComponentsBuilder()
 
     override fun getKotlinClassFor(objCClassOrProtocol: ObjCClassOrProtocol, isMeta: Boolean): Classifier {
-        return declarationMapper.getKotlinClassFor(objCClassOrProtocol, isMeta)
+        return declarationMapper.getKotlinClassFor(objCClassOrProtocol, isMeta, (nativeIndex is J2ObjCNativeIndex))
```

Comment:
Can we do this without using is.  Using is (the Kotlin version of instance of) is a code smell that something isn't quite right.

Reply:
Good idea, replaced with a language check from the configuration